src/main/resources/migration/common/V17__migrate_com_details_to_new_table.sql

```diff
@@ -0,0 +1,47 @@
+-------------------------------------------------------
+----- Create new community_offender_manager table -----
+-------------------------------------------------------
+CREATE TABLE community_offender_manager
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    staff_identifier INTEGER NOT NULL UNIQUE,
+    username VARCHAR(40) NOT NULL UNIQUE,
+    email VARCHAR(200),
+    created_timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    last_updated_timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+-----------------------------------------------------------------------
+----- Create new community_offender_manager_licence mapping table -----
+-----------------------------------------------------------------------
+CREATE TABLE community_offender_manager_licence_mailing_list
+(
+    id SERIAL NOT NULL PRIMARY KEY,
+    community_offender_manager_id INTEGER NOT NULL REFERENCES community_offender_manager(id),
+    licence_id INTEGER NOT NULL REFERENCES licence(id),
+    created_timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    last_updated_timestamp TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP
+);
+
+---------------------------------------------------------------------
+--- Drop redundant columns in licence table -----
+---------------------------------------------------------------------
+ALTER TABLE licence DROP COLUMN com_first_name;
```

Comment:
Multiple ALTER commands because H2 does not support dropping columns in bulk

Reply:
We could probably drop the DB and consolidate all these migrations at some point? Later though.